### PR TITLE
Credentials utility for the PSC namespace in cbapi

### PR DESCRIPTION
**Pull request by zacharyestep**
See #149

### CODE REVIEW

File: bin/cbapi-psc [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        print("An existing credential file exists at {0}.".format(credential_file))
+        resp = input("Do you want to continue and overwrite the existing configuration? [Y/N] ")
+        if resp.strip().upper() != "Y":
+            print("Exiting.")
+            return 1
+
+    if not os.path.exists(credential_path):
+        os.makedirs(credential_path, 0o700)
+
+    url = input("URL to the Cb Defense API server (do not include '/integrationServices') [https://hostname]: ")
```
Comment (jgarman) [inline bin/cbapi-psc]: Should be "PSC" here

File: docs/getting-started.rst [hunk 2]
```diff
@@ -25,7 +25,7 @@ CB Response or CB Protection servers inside the API credential file and select w
 at runtime.
 
 To create the initial credential file, a simple-to-use script is provided. Just run the ``cbapi-response``,
-``cbapi-protection``, or ``cbapi-defense`` script with the ``configure`` argument. On Mac OS X and Linux::
+``cbapi-protection``,``cbapi-psc`` or ``cbapi-defense`` script with the ``configure`` argument. On Mac OS X and Linux::
```
Comment (jgarman) [inline docs/getting-started.rst]: let's just remove `cbapi-defense` as part of this... the `cbapi.psc.defense` module should work with the `credentials.psc` file - double check this of course

**Comment by smultani** [comment]
@zacharyestep @jgarman 
how does this psc utility handle org-key?
Also some psc routes go to api-prod01 and some go to defense-prod01. How do we handle that?

**Comment by zacharyestep** [comment]
Closing ...

**Pull request closed without merging**